content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Synchronized file sharing
+description: Get started with Synchronized file sharing on Docker Desktop.
+sitemap: false
+---
+
+> **Early Access**
+>
+> Synchronized file sharing is an early-access feature. 
+>
+> If you want to test out Synchronized file sharing, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{ .restricted }
+
+Synchronized file sharing is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
+
+![Image of Synchronized file shares pane](images/synched-file-sharing.png)
+ 
+## Who is it for?
+
+For developers who: 
+- Have large repositories or monorepos with 100 000 files or more totaling hundreds of megabytes or even gigabytes
+- Are using virtual filesystems, such as VirtioFS, gRPC FUSE, and osxfs, which are no longer scaling well with their codebases 
+- Regularly encounter performance limitations
+- Don't want to worry about file ownership or spend time resolving conflicting file-ownership information when modifying multiple containers
+
+## How does Synchronized file sharing work?
+
+A Synchronized file share behaves just like a virtual file share, but takes advantage of Mutagen's high-performance, low-latency code synchronization engine to create a synchronized cache of the host files on an ext4 filesystem within the Docker Desktop VM. If you make filesystem changes on the host or in the VM’s containers it propagates via bidirectional synchronization.
+
+Synchronized file sharing also adds an additional Docker Context, `desktop-linux-mutagen`, which uses Docker socket middleware to automatically replace bind mounts with in-VM caches whenever possible.
+
+After creating a file share instance, any container using a bind mount that references a host filesystem location which matches the stated synchronized file share host location, is provided by the Synchronized file sharing feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem bind-mounting mechanism, for example VirtioFS or gRPC-FUSE.
+
```

Comment:
We could add something here showing before/after Mutagen speed comparison? 
Either get users to run `time git status` like in Jacob's demos, or with a simple graph?

Reply:
We have some benchmarks that we prepared for DockerCon: https://docs.google.com/spreadsheets/d/1eND7rSHF8USorsG0-qPNOyCGIbsrgDsude0tqWacJVo/edit#gid=0

Maybe a table of those?  @smnovick what do you think?